### PR TITLE
Added dynamic social image

### LINKED ISSUE #276: [qwik-pod] 2. Add social image and description
# Add framework social image and description

## Background

As an user I would like to have the possibility to share the site on social media and for it to have a nice image and description defined, so that it will increase its visibility and shareability

## Acceptance

- [ ] Add metadata description
- [ ] Add social image
- [ ] Test the site on "https://socialsharepreview.com/"

### PR DESCRIPTION
**Pull request by AllStackDev1**
## Type of change

- [ ] Content addition
- [ ] Bug fix
- [x] Behavior change

## Summary of change
Added dynamic social image. This is still in beta mode to determine if it works as intended.
There is no preview link to know if this actually works, till it is merged into the main branch.

## Checklist

- [x] This change resolves #276
- [x] I have confirmed with a maintainer that this change is desired and been
      given the go-ahead to work on it in the relevant issue discussion
- [ ] The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
- [ ] I have verified the change and the rest of the site works as expected
- [ ] If the change introduces new components, these have been added to
      `packages/system/src/components` with a `.stories.tsx` file that
      adequately renders possible variations of each component.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,12 +15,16 @@
...
 		"dev:qwik": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=qwik yarn workspace @framework/site dev",
+		"dev:solidjs": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=solidjs yarn workspace @framework/site dev",
+		"dev:svelte": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=svelte yarn workspace @framework/site dev",
 		"build:system": "yarn workspace @framework/system build",
 		"build:landing": "yarn workspace @framework/landing-site build",
 		"build:react": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=react yarn workspace @framework/site build",
 		"build:angular": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=angular yarn workspace @framework/site build",
 		"build:vue": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=vue yarn workspace @framework/site build",
 		"build:qwik": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=qwik yarn workspace @framework/site build",
+		"build:solidjs": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=solidjs yarn workspace @framework/site build",
```
Comment (honzikec) [inline package.json:26]: The `solidjs` and `svelte` part may be a mistake? 🙂

Comment (AllStackDev1) [reply package.json:26]: hmmm, I did a git rebase of the main branch, seems there are other teams working on solidjs.

Comment (honzikec) [reply package.json:26]: Interesting, maybe you need to do an interactive rebase and drop the solidjs/svelte commits...

File: packages/site/src/layouts/base.astro [hunk 2]
```diff
@@ -20,9 +20,14 @@ const navData = searchData.map(o => pick(o, ["name", "subCategories"]))
 <html class={themes[`${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}Theme`]} lang="en">
   <head>
 		{fontHeadTags}
-    <title>{`${title} | ${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}.framework.dev`}</title>
+    	<title>{`${title} | ${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}.framework.dev`}</title>
```
Comment (honzikec) [inline packages/site/src/layouts/base.astro:23]: Nitpick: the indentation in this file seems a bit strange 🙂

**Review by honzikec — APPROVED** [review]
Approved with a small nitpick 🙂

**Review by honzikec — APPROVED** [review]
LGTM

File: packages/site/src/layouts/base.astro (hunk 2 above)
Comment (AllStackDev1) [reply packages/site/src/layouts/base.astro:23]: I believe it doesn't respect linting at this point, on vscode, it is formatted nicely but on GitHub, the indentation for the title shoots off.

**Pull request merged**